tests/scancode/test_cli.py

```diff
@@ -452,3 +452,25 @@ def test_scan_can_run_from_other_directory():
         print(stderr)
     assert rc == 0
     check_scan(test_env.get_test_loc(expected_file), result_file, strip_dates=True, regen=False)
+
```

Comment:
You need to add proper unit tests to your function alone and not only CLI integration tests